crates/contract/tests/sandbox/common.rs

```diff
@@ -81,9 +80,7 @@ pub const GAS_FOR_VOTE_RESHARED: Gas = Gas::from_tgas(15);
 /// This is the current deposit required for a contract deploy. This is subject to change but make
 /// sure that it's not larger than 2mb. We can go up to 4mb technically but our contract should
 /// not be getting that big.
-///
-/// TODO(#771): Reduce this to the minimal value possible after #770 is resolved
```

Comment:
why removing the comment linking to the issue? #771 is still open

Reply:
Yeah seems like we should keep it. I restored it when I rebased.